plugins/node/plugin.zsh

```diff
@@ -0,0 +1,18 @@
+# Color definitions
+GEOMETRY_COLOR_NODE_NPM_VERISON=${GEOMETRY_COLOR_NODE_NPM_VERISON:-black}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_NODE_NPM_VERISON=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
+GEOMETRY_NODE_NPM_VERISON=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERISON GEOMETRY_SYMBOL_NODE_NPM_VERISON) 
+
+geometry_prompt_node_setup() 
```

Comment:
Should be `geometry_prompt_node_setup() {}` in order to define the function correctly.